java/ql/lib/ext/net.schmizz.sshj.model.yml

```diff
@@ -0,0 +1,12 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["net.schmizz.sshj", "SSHClient", False, "auth", "", "", "Argument[0]", "credentials-username", "hq-generated"]
```

Comment:
This model and about 13 others also don't seem to be in the original SensitiveApi.qll file. Should they be removed from this PR as well?
```suggestion
```

Reply:
Ok I've gone through `otherApiCallableCredentialParam` and I believe I have removed all of the extra models.